scripts/card.js

```diff
@@ -1,113 +0,0 @@
-import { grid, section } from "./variables.js";
-
-//function to create a card for each book
-export const createCard = (book) => {
-  //create a container for the whole book info
-  let card = document.createElement("div");
-  card.classList = "card";
-
-  //add a title
-  let elementBookTitle = document.createElement("h2");
-  elementBookTitle.appendChild(document.createTextNode(book.title));
-  elementBookTitle.classList = "card__book-title";
-
-  //add an author
-  let elementBookAuthor = document.createElement("h3");
-  elementBookAuthor.appendChild(document.createTextNode(book.author));
-  elementBookAuthor.classList = "card__book-author";
-
-  //add an image
-  let elementBookImage = document.createElement("img");
-  elementBookImage.src = book.image;
-  elementBookImage.classList = "card__book-image";
-
-  //add description modal
-  let cardDesc = document.createElement("div");
-  cardDesc.classList = "desc";
-
-  //add text to description modal
-  let elementBookDescription = document.createElement("p");
-  elementBookDescription.appendChild(document.createTextNode(book.description));
-
-  //create description modal close button
-  const closeModalBtn = document.createElement("button");
-  closeModalBtn.appendChild(document.createTextNode("Close Description"));
-  closeModalBtn.classList = "close";
-
-  //append description and close button to description modal
-  cardDesc.append(elementBookDescription);
-  cardDesc.append(closeModalBtn);
-  section.append(cardDesc);
-
-  //add button to open description modal
-  let btnDesc = document.createElement("button");
-  btnDesc.classList = "card__Btn";
-  btnDesc.appendChild(document.createTextNode("Description"));
-
-  //event to open description modal when DESCRIPTION button clicked // SHOULD EVENT BE HERE OR SOMEWHERE ELSE?
-  btnDesc.addEventListener("click", () => {
-    cardDesc.classList.add("desc--open");
-  });
-
-  //close description button when CLOSE button clicked // SHOULD EVENT BE HERE OR SOMEWHERE ELSE?
-  closeModalBtn.addEventListener("click", () => {
-    cardDesc.classList.remove("desc--open");
-  });
```

Comment:
This creates a lot of modals in your DOM that may never be opened. It might make more sense to have one modal and load the correct content into it when you click open. It's actually a lot easier to do that in react so maybe leave for a future refactor

Reply:
Leave for now, but think about in react